### PR TITLE
fix: enable production url for basepath

### PR DESCRIPTION
**Pull request by xke**

### CODE REVIEW

**Comment by xke** [comment]
added `package.json` so `npm` can be used on the `movies` example

File: examples/playground-movies/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "playground-movies-example",
```
Comment (viveknair) [inline examples/playground-movies/package.json:2]: https://github.com/gentrace/gentrace-node/blob/f2adddcad40b13d1b92b90408f5e244d87290f6a/examples/files/package.json#L2

We normally follow the convention `@examples/playground-movies`

File: examples/playground-movies/helpers.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import OpenAI from "openai";
 init({
   apiKey: process.env.GENTRACE_API_KEY ?? "",
   runName: "Example run for Playground SDK",
-  basePath: "http://localhost:3000/api",
+  basePath: process.env.GENTRACE_BASE_PATH ?? "",
```
Comment (viveknair) [inline examples/playground-movies/helpers.ts:9]: We normally don't tell users to configure `basePath`, so `GENTRACE_BASE_PATH` might be empty a lot. Does it fall back to prod if `basePath` == ""?

**Review by viveknair — CHANGES_REQUESTED** [review]

File: examples/playground-movies/helpers.ts (hunk 2 above)
Comment (xke) [reply examples/playground-movies/helpers.ts:9]: yes, that's what i saw

**Review by viveknair — APPROVED** [review]

**Pull request merged**